tests/unit/-private/properties/blurrable-test.js

```diff
@@ -171,56 +153,62 @@ moduleForProperty('blurrable', function(test) {
       }
     });
 
-    this.adapter.createTemplate(this, page);
+    await this.adapter.createTemplate(this, page);
 
-    this.adapter.throws(assert, function() {
+    await this.adapter.throws(assert, function() {
       return page.foo.bar.baz.qux();
     }, /page\.foo\.bar\.baz\.qux/, 'Element not found');
   });
 
-  test('Does not raise error when blurring focusable elements', function(assert) {
-    assert.expect(0);
-
-    let page = create({
-      foo: {
-        bar: {
-          input: blurrable('input'),
-          select: blurrable('select'),
-          a: blurrable('a'),
-          area: blurrable('area'),
-          iframe: blurrable('iframe'),
-          button: blurrable('button'),
-          contentEditable: blurrable('[contenteditable]'),
-          tabindex: blurrable('[tabindex]'),
+  if (adapter === 'integration' || adapter === 'acceptance') {
+    test('Does not raise error when blurring focusable elements', async function(assert) {
+      assert.expect(0);
+
+      let page = create({
+        foo: {
+          bar: {
+            input: blurrable('input'),
+            select: blurrable('select'),
+            a: blurrable('a'),
+            area: blurrable('area'),
+            iframe: blurrable('iframe'),
+            button: blurrable('button'),
+            contentEditable: blurrable('[contenteditable]'),
+            tabindex: blurrable('[tabindex]'),
 
+          }
         }
-      }
-    });
-
-    this.adapter.createTemplate(this, page, `
-      <input/>
-      <a href="foo"></a>
-      <area href="foo"></a>
-      <iframe></iframe>
-      <select></select>
-      <button></button>
-      <div contenteditable></div>
-      <div tabindex=-1></div>
-    `);
-
+      });
 
-    page.foo.bar.input();
-    page.foo.bar.select();
-    page.foo.bar.a();
-    page.foo.bar.area();
-    page.foo.bar.iframe();
-    page.foo.bar.button()
-    page.foo.bar.contentEditable();
-    page.foo.bar.tabindex();
-  });
+      await this.adapter.createTemplate(this, page, `
+        <input/>
+        <a href="foo"></a>
+        <area href="foo"></a>
+        <iframe></iframe>
+        <select></select>
+        <button></button>
+        <div contenteditable></div>
+        <div tabindex=-1></div>
+      `);
+
+
+      page.foo.bar.input();
+      page.foo.bar.select();
+      page.foo.bar.a();
+      page.foo.bar.area();
+      page.foo.bar.iframe();
+      page.foo.bar.button()
+      page.foo.bar.contentEditable();
+      page.foo.bar.tabindex();
+    });
+  }
 
-  test('raises an error when the element is not focusable', function(assert) {
-    assert.expect(4);
+  test('raises an error when the element is not focusable', async function(assert) {
+    if (adapter === 'acceptance' || adapter === 'integration') {
```

Comment:
Personally, I wouldn't use `assert.expect()` in this case since the only way it would fail is if a promise rejected, which would fail the test anyway. But I don't feel comfortable applying that preference to code written so recently, unless @pzuraq agrees that removing it makes sense.

Reply:
I did it mostly out of habit, with the older style tests it was very easy to have flakiness because assertions would get dropped in race conditions. I don't think there's a reason to keep it at this point.